src/components/TypingIndicator/__tests__/TypingIndicator.test.js

```diff
@@ -139,4 +139,120 @@ describe('TypingIndicator', () => {
 
     expect(tree).toMatchInlineSnapshot(`null`);
   });
+
+  it('should render TypingIndicator if user is typing in thread', async () => {
+    const client = await getTestClientWithUser();
+    const ch = generateChannel({ config: { typing_events: true } });
+    useMockedApis(client, [getOrCreateChannelApi(ch)]);
+    const channel = client.channel('messaging', ch.id);
+    await channel.watch();
+
+    const { container } = render(
```

Comment:
let's pull the render logic out of the tests and put in the beginning of the file and name it something like renderTypingIndicator and pass values needed for each test into it, so don't need to repeat this logic so much